ONSdigital/response-operations-ui
# rOps improvements - Survey Collection Exercise pages redesign

# What and why?
First redesign of SEFT and EQ collection exercise page
# How to test?
Deploy this pr to your environment and set up and set to live both an eQ and SEFT survey. Make sure it all looks right and you're able to do everything you're supposed to, once its live you should only be able to edit the reference information and employment. Also check that without survey edit permissions you aren't able to edit a collection exercise.
# Trello
https://trello.com/c/MGK6iJzw/1601-rops-improvements-survey-collection-exercise-pages-redesign
